bladeui/css/blade/referralsForm.scss

```diff
@@ -36,4 +36,36 @@
       outline: none;
     }
   }
+
+  .loader {
```

Comment:
please take away loader to global styles,seems like it will be used not only here